geode-core/src/main/java/org/apache/geode/internal/cache/persistence/PersistenceAdvisorImpl.java

```diff
@@ -846,6 +846,7 @@ public void finishPendingDestroy() {
           if (previousAdvice != null && !previousAdvice.getReplicates().isEmpty()) {
             logger.info(
                 LocalizedMessage.create(LocalizedStrings.PersistenceAdvisorImpl_RETRYING_GII));
+            Thread.sleep(200);
```

Comment:
Why was this Thread.sleep added?

Reply:
We saw a hot loop in testing that was spitting out hundreds of MB of log messages.

Reply:
Let's remove this and file a separate ticket.